gradle/lib-admin.gradle

```diff
@@ -20,19 +20,39 @@ def applyExcludedTasks()
     gradle.includedBuild( 'lib-admin-ui' ).getLoadedSettings().getStartParameter().setExcludedTaskNames( excludedTasks )
 }
 
+def applyBuildTaskDependencies()
+{
+    def libAdminBuildTask = gradle.includedBuild( 'lib-admin-ui' ).task( ':build' )
+
+    if ( hasTask( 'copyDevResources' ) ) {
+        copyDevResources.dependsOn libAdminBuildTask
+    }
+    if ( hasTask( 'typescript' ) ) {
```

Comment:
It looks like overkill to have typescript and `webpack` depend on libadminui build.
Both tasks depend on `npmInstall`, but `npmInstall` always depends on `copyDevResources`